procurement_mrp_no_confirm/i18n/procurement_mrp_no_confirm.pot

```diff
@@ -0,0 +1,32 @@
+# Translation of Odoo Server.
```

Comment:
@pedrobaeza just to confirm, next time i move module from odoomrp i should remove .pot file?